src/data-workspace/inputs/option-set.js

```diff
@@ -42,10 +41,19 @@ export const OptionSet = ({
         }
     }
 
-    const optionSet = selectors.getOptionSetById(metadata, optionSetId)
+    const optionSet = useOptionSet(optionSetId)
+
+    if (!optionSet) {
+        console.warn('Could not find optionSet with id:', optionSetId)
```

Comment:
I'm wondering: Do we have an official policy whether we want to show console output in production code?